community/server/src/test/resources/META-INF/services/org.neo4j.server.plugins.PluginLifecycle

```diff
@@ -0,0 +1 @@
+org.neo4j.server.modules.ExtensionInitializerTest$PropertyCollectorPlugin
```

Comment:
That is for `ExtensionInitializerTest`. `ExtensionInitializer` uses `Service.load( PluginLifecycle.class )` to load plugins and that is a way how those will be found.